boundary/analyze.py

```diff
@@ -145,22 +147,51 @@ def lookup_if_global(name_and_file):
     file = global_fn_dict.get(name, None) if file == '?' else file
     return (name, file) if file else None
 
+def sidecar_inflect(sidecar, in_vmlinux):
+    assert len(sidecar - in_vmlinux) == 0, \
+            'sidecar functions should not be optimzied by GCC'
+
+    insiders = set()
```

Comment:
The term `insider` means the function is updated implicitly, right after all interface function are redirected.

However this is not the case for these descendants of sidecar functions. Replacing one sidecar function doesn't guarantee its descendants gets replaced as well.

So a better name is needed here.

Reply:
rename to `leftover`